### PR TITLE
[CA-1987] ETQ user je veux lister les seconds facteurs disponibles sur mon profil

### PR DESCRIPTION
**Pull request by moust**
[JIRA](https://reach5.atlassian.net/browse/CA-1987)

Sandbox : [https://github.com/ReachFive/reachfive-sandbox/pull/98](https://github.com/ReachFive/reachfive-sandbox/pull/98)

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -72,6 +73,10 @@ export class UiClient {
         this._showWidget(mfaCredentialsWidget, options);
     }
 
+    showMfaList(options) {
```
Comment (axuan25) [inline src/client.js:76]:
```suggestion
    showMfaCredendials(options) {
```

File: src/widgets/mfa/mfaListWidget.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import compose from '@hypnosphi/recompose/compose';
+
+import { Info } from '../../components/miscComponent';
+import { Card } from '../../components/form/cardComponent';
+import { createWidget } from "../../components/widget/widget";
```
Comment (axuan25) [inline src/widgets/mfa/mfaListWidget.jsx:6]:
```suggestion
import { createWidget } from '../../components/widget/widget';
```

**Review by axuan25 — APPROVED** [review]

**Review by thomasreach5 — APPROVED** [review]

File: src/client.js (hunk 1 above)
Comment (thomasreach5) [reply src/client.js:76]: Don't commit this through Github, there's a typo.

Comment (moust) [reply src/client.js:76]: 🕵️

**Comment by JonathanM90** [comment]
il ne manquerait pas ?
`showMfaCredentials(options: any): Promise<void>`

dans le fichier index.d.ts

pour avoir 
`export interface Client {
    showAuth(options: any): Promise<void>,
    showEmailEditor(options: any): Promise<void>,
    showPasswordEditor(options: any): Promise<void>,
    showPhoneNumberEditor(options: any): Promise<void>,
    showPasswordReset(options: any): Promise<void>,
    showPasswordless(options: any): Promise<void>,
    showProfileEditor(options: any): Promise<void>,
    showSocialAccounts(options: any): Promise<void>,
    showSocialLogin(options: any): Promise<void>,
    showWebAuthnDevices(options: any): Promise<void>,
    showMfaCredentials(options: any): Promise<void>
}`

**Comment by moust** [comment]
> il ne manquerait pas ? `showMfaCredentials(options: any): Promise<void>`
> 
> dans le fichier index.d.ts
> 
> pour avoir `export interface Client { showAuth(options: any): Promise<void>, showEmailEditor(options: any): Promise<void>, showPasswordEditor(options: any): Promise<void>, showPhoneNumberEditor(options: any): Promise<void>, showPasswordReset(options: any): Promise<void>, showPasswordless(options: any): Promise<void>, showProfileEditor(options: any): Promise<void>, showSocialAccounts(options: any): Promise<void>, showSocialLogin(options: any): Promise<void>, showWebAuthnDevices(options: any): Promise<void>, showMfaCredentials(options: any): Promise<void> }`

Bien vu. Je pensais que ce fichier était généré au build mais il semble que non.

**Pull request merged**